tiledb/sm/query/readers/reader_base.cc

```diff
@@ -1008,6 +1008,10 @@ tuple<uint64_t, uint64_t> ReaderBase::compute_chunk_min_max(
     const uint64_t num_chunks,
     const uint64_t num_range_threads,
     const uint64_t thread_idx) const {
+  if (num_chunks == 0) {
```

Comment:
Let's go ahead and add a unit test for this function so we can test more permutations without needing a full array and query.